client/index.html

```diff
@@ -9,14 +9,22 @@
   <script type="text/javascript" src="../shared/forge.min.js"></script>
   <script type="text/javascript" src="../shared/mpc.js"></script>
   <script type="text/javascript" src="style/bootstrap/js/bootstrap.js"></script>
+  <script type="text/javascript" src="style/alertify/alertify.js"></script>
```

Comment:
@frederickjansen - thanks for the feedback. I made changes that should address your comments. Let me know if anything else needs to be changed.